site/src/pages/[shortname]/[blockSlug].page.tsx

```diff
@@ -164,6 +164,26 @@ const BlockPage: NextPage<BlockPageProps> = ({
     return catalog.filter(({ name }) => name !== blockMetadata.name);
   }, [catalog, blockMetadata]);
 
+  const repositoryUrl = useMemo(() => {
+    const { type, url, directory } = blockMetadata.repository ?? {};
+
+    if (url) {
+      let returnUrl = url;
+
+      if (type === "git") {
+        returnUrl = returnUrl.replace(".git", "");
```

Comment:
This'd be better with a RegExp that only replaced `.git` at the end of the line - this will mangle URLs like `"https://www.github.com/blockprotocol/blockprotocol"`

Reply:
Done in the latest commit: https://github.com/blockprotocol/blockprotocol/pull/198/commits/f7583700a9309bc5ccf963208524f07f15dfd9d5